data/presets/playground/funnel_ball.json

```diff
@@ -0,0 +1,10 @@
+{
+    "icon": "maki-playground",
+    "geometry": [
+        "point"
```

Comment:
By the way, I think the reason this is a point rather than an area is that this tag represents just the funnel; the funnel ball court is commonly tagged `leisure=pitch`. The same seems to be true of tetherball, for which we’ve added a pitch/sport preset. Should we make that distinction clear? Does it matter?

Reply:
Good idea, I'll add presets for `sport=funnel_ball` and `playground=tetherball`